lib/mixpanel-node.js

```diff
@@ -132,6 +132,15 @@ var create_client = function(token, config) {
         metrics.send_request(endpoint, data, callback);
     };
 
+    var parse_time = function(time) {
+        if (time === void 0) {
```

Comment:
Is there a way to avoid (ha) this void 0? I get it's for robustness, but it seems dirty. What if you did
```
        if (Object.prototype.toString.call(time) === '[object Date]') {
            time = Math.floor(time.getTime() / 1000);
        }
        if (!time && time !== 0) {
            throw new Error("Import methods require you to specify the time of the event");
        }
        return time;
```
Seems like that would handle all the type of input we want. (also know you didn't write the void 0 so feel free to ignore)